flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectQueryITCase.java

```diff
@@ -469,7 +469,14 @@ public void testInsertDirectory() throws Exception {
                 .await();
         java.nio.file.Path[] files =
                 FileUtils.listFilesInDirectory(
-                                Paths.get(dataDir), (path) -> !path.toFile().isHidden())
+                                Paths.get(dataDir),
+                                (path) ->
+                                        !path.toFile().isHidden()
+                                                && !path.toFile()
+                                                        .getName()
+                                                        .equals(
+                                                                HiveOptions
+                                                                        .SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME))
```

Comment:
Should be `HiveOptions.SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME.defaultValue()` or `tableEnv.getConfig().get(HiveOptions.SINK_PARTITION_COMMIT_SUCCESS_FILE_NAME)`.